### DISCUSSION TITLE
Setup github issue templates

### DISCUSSION THREAD

**Discussion by markpeek** [ghostty-org/ghostty#3221] [Feature Requests, Ideas] [answered]
It is great to see ghostty 1.0 out and the enthusiasm from people opening issues for bugs and features. Given the ghostty project uses discussions for feature requests and behavior changes, setting up [github issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) would help guide new users and contributors on what is needed for bug reporting vs. feature requests. This is likely best done through the github Settings by an owner for the initial pass rather than a PR approach.

**Comment by 00-kat** [comment, marked as answer]
Very late, but for future visitors:
- Mitchell ended up doing this at some point.
- For *Discussion* templates there's now #7012; us helpers were discussing this privately on Discord for very long and it only very recently got moved to a proper Pull Request.
